docs/mnist_example/requirements.txt

```diff
@@ -1,2 +1,2 @@
-torch
-torchvision
+torch>=1.9.0
```

Comment:
We don't know what exact versions are required. The versions were used in an online article (https://analyticsindiamag.com/guide-to-feed-forward-network-using-pytorch-with-mnist-dataset/) Since we are specifying the minimal versions, lower versions are preferred since they are less restrictive.

Reply:
Sure. torch>=1.9.1 and torchvision>=0.10.1 are fine.

Reply:
updating to these values